docs/_basic/acknowledging_actions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Acknowleding actions
```

Comment:
This misses the point that everything besides events need to be acknowledged (rather than actions specifically).